src/main/java/hudson/plugins/sloccount/SloccountBuildAction.java

```diff
@@ -101,11 +115,33 @@ SloccountBuildAction getPreviousAction(){
         return null;
     }
 
-    public AbstractBuild<?,?> getBuild(){
+    public Run<?,?> getBuild(){
```

Comment:
It breaks the binary compatibility. But it's up to the plugin maintainer

Reply:
It's still a breaking change. So I would advice a major version update in this case